clubs/admin.py

```diff
@@ -128,6 +128,20 @@ def club(self, obj):
         return obj.club.name
 
 
+class ProfileAdmin(admin.ModelAdmin):
+    search_fields = ('user__username', 'user__email')
+    list_display = ('user', 'email', 'graduation_year', 'studies')
+    list_filter = ('graduation_year', 'school', 'major')
+
+    def email(self, obj):
+        return str(obj.user.email or None)
+
+    def studies(self, obj):
+        major = ', '.join(obj.major.values_list('name', flat=True))
+        school = ', '.join(obj.school.values_list('name', flat=True))
+        return '{} - {}'.format(school or None, major or None)
```

Comment:
Instead of `None` for these might be able to say `'Undeclared'`

Reply:
There are actually three cases where this can happen, either the user hasn't filled out the user profile form yet, if they intentionally leave it blank, or if they actually don't have a major.

`None` isn't too great, but the placeholder should represent the possibility of all three of these cases.

Also note that this will only be visible to admins!

Reply:
yepo